install_config/aggregate_logging.adoc

```diff
@@ -937,6 +949,63 @@ existing `journald` log driver configuration. It is recommended to use the
 Logging Driver] for instructions to change your existing log driver
 configuration to `json-file`.
 
+[[fluentd-file]]
+*Configuring Fluentd to Send Logs to a File*
+
+Fluentd writes logs to a file specified by the `LOGGING_FILE_PATH` environment variable, by default `/var/log/fluentd/fluentd.log`. 
+
+Use the *logs* utility to print out the contents of Fluentd log files. For example:
+
+----
+oc exec <pod> logs <1>
+----
+
+<1> Specify the name of the Fluentd or Elasticsearch pod.
+
+For example:
+
+----
+oc exec logging-es-data-master-2-deploy logs
+oc exec logging-kibana-1 logs
+----
+
+The contents of log files are printed out, starting with the oldest log.  Use `-f` option to follow what is being written into the logs.
+
+To control the size of the logs, you can specify a maximum log size and the length of time that {product-title} retains the logs. 
+ 
+* The `LOGGING_FILE_SIZE` parameter specifies how large the file can be before {product-title} rotates the files. When the log reaches the specified size, {product-title} automatically compresses and renames the *fluentd.log* log file so that new logging data can be collected.  
```

Comment:
> {product-title} automatically compresses and renames

There is no compression, OpenShift just renames the files. It is possible to compress them but we don't do it.